### DISCUSSION TITLE
What is the benefit of this package before `json_serializable`?

### DISCUSSION THREAD

**Discussion by bambinoua** [k-paxian/dart-json-mapper#224] [Q&A]
Why do I need this package if there is `json_serializable`?

**Comment by k-paxian** [comment]
You don't need it in general, but you can consider. What DevEx you have with `json_serializable`? and make a decision is it a good fit for your use cases.

Usually people are frustrated when `json_serializable` doesn't support some use case they really need, and they just left with the big **NO** from the maintainers, that's why imho `json_serializable` has 10x more open issues.
An example issue specially for you 😄  [issue](https://github.com/google/json_serializable.dart/issues/853)

![image](https://github.com/k-paxian/dart-json-mapper/assets/18071010/d56cec2b-f1ae-4a01-a67d-40274403289c)


Here is a brief comparison of different approaches
https://github.com/k-paxian/dart-json-mapper/tree/master/perf-test/test/starlink
